.sonarqube/sonar-scanner.properties

```diff
@@ -0,0 +1,42 @@
+sonar.projectKey=usdot-fhwa-stol_carma-msgs
+sonar.organization=usdot-fhwa-stol
+sonar.cfamily.cache.enabled=false
+sonar.cfamily.compile-commands=/opt/carma/build/compile_commands.json
+sonar.cfamily.threads=4
+sonar.host.url=https://sonarcloud.io
+#sonar.sources=src/main
+sonar.cfamily.gcov.reportsPath=/opt/carma/coverage_reports/gcov
+sonar.coverageReportPaths=/opt/carma/coverage_reports/gcov/coverage.xml
+# Set Git as SCM sensor
+sonar.scm.disabled=false
+sonar.scm.enabled=true
+sonar.scm.provider=git
+sonar.sourceEncoding=UTF-8
+sonar.c.file.suffixes=-
+sonar.cpp.file.suffixes=.cpp,.h,.tpp
+# Modules, C++ packages
+sonar.modules= carma_debug_msgs, \
+  carma_debug_ros2_msgs, \
+  carma_driver_msgs, \
+  carma_localization_msgs, \
+  carma_msgs, \
+  carma_perception_msgs, \
+  carma_planning_msgs, \
+  carma_v2x_msgs, \
+  cav_msgs, \
+  cav_srvs, \
+  j2735_msgs, \
+  j2735_v2x_msgs
+
+carma_debug_msgs.sonar.projectBaseDir                  = /opt/carma/src/carma-msgs/carma_debug_msgs
+carma_debug_ros2_msgs.sonar.projectBaseDir             = /opt/carma/src/carma-msgs/carma_debug_ros2_msgs
+carma_driver_msgs.sonar.projectBaseDir     = /opt/carma/src/carma-msgs/carma_driver_msgs
+carma_localization_msgs.sonar.projectBaseDir      = /opt/carma/src/carma-msgs/carma_localization_msgs
+carma_msgs.sonar.projectBaseDir                     = /opt/carma/src/carma-msgs/carma_msgs
+carma_perception_msgs.sonar.projectBaseDir         = /opt/carma/src/carma-msgs/carma_perception_msgs
+carma_planning_msgs.sonar.projectBaseDir            = /opt/carma/src/carma-msgs/carma_planning_msgs
+carma_v2x_msgs.sonar.projectBaseDir                = /opt/carma/src/carma-msgs/carma_v2x_msgs
+cav_msgs.sonar.projectBaseDir            = /opt/carma/src/carma-msgs/cav_msgs
+cav_srvs.sonar.projectBaseDir             = /opt/carma/src/carma-msgs/cav_srvs
+j2735_msgs.sonar.projectBaseDir      = /opt/carma/src/carma-msgs/j2735_msgs
+j2735_v2x_msgs.sonar.projectBaseDir   = /opt/carma/src/carma-msgs/j2735_v2x_msgs

```

Comment:
Similar to previous comment, can we add the projectBaseDir for can_msgs, j3224_v2x_msgs, and j3224_msgs as well?